### PR TITLE
Don't show errors for kernel usages

### LINKED ISSUE #475: Handle kernel modules
We should handle kernel modules without just throwing errors. Possibly we could even parse the js files to have go to definitions. We also need to check how the compiler handles them with type inference.

### PR DESCRIPTION
**Pull request by jmbockhorst**
Fixes #475.

### CODE REVIEW

**Review by Razzeee — CHANGES_REQUESTED** [review]
Small nitpick

File: src/util/elmUtils.ts [hunk 1]
```diff
@@ -389,3 +390,16 @@ export function flattenExposedModules(
 
   return Object.values(exposedModules).reduce((a, b) => a.concat(b), []);
 }
+
+export function nameIsKernel(name: string): boolean {
+  return name.startsWith("Elm.Kernel.");
+}
+
+export function isKernelProject(project: ElmProject): boolean {
```
[the remark this answers is not in the document]
Comment (jmbockhorst) [reply src/util/elmUtils.ts:398]: I can't think of a better name.

[the remark this answers is not in the document]
Comment (jmbockhorst) [reply src/util/elmUtils.ts:398]: The compiler uses `Name.isKernel` and `Pkg.isKernel`.

[the remark this answers is not in the document]
Comment (Razzeee) [reply src/util/elmUtils.ts:398]: Would have thought `isPossibleKernelProject`, but if the compiler is using the same wording, it's fine for me

**Comment by harrysarson** [comment]
Thanks @Razzeee and @jmbockhorst 🎉

**Pull request merged**